package.json

```diff
@@ -3,36 +3,111 @@
   "version": "0.1.0",
   "description": "%CUSTOM_PLUGIN_SERVICE_DESCRIPTION%",
   "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "scripts": {
+    "prepare": "node ./scripts/prepare.js",
+    "lint": "eslint . --ignore-path=.gitignore --ext=ts",
+    "lint:fix": "npm run lint -- --fix",
+    "unit": "jest",
+    "test": "npm run lint && npm run unit",
     "coverage": "npm run unit -- --coverage",
-    "lint": "eslint . --ignore-path=.gitignore",
+    "clean": "rm -rf ./dist",
+    "check-types": "tsc --pretty --project tsconfig.json",
+    "compile": "npm run check-types && swc src -d dist",
+    "build": "npm run clean && npm run compile",
+    "dev": "swc src -d dist -w",
```

Comment:
Can you provide a brief explanation on how to use these new commands in the README?

Reply:
https://github.com/danibix95/Typescript-LC39-Template/blob/master/README.md?plain=1#L154-L173